Cargo.toml

```diff
@@ -19,7 +19,7 @@ alga = "0.9"
 nalgebra = "0.22"
 rand = "0.7"
 rayon = "1.2"
-lazy_static = "1.4"
+once_cell = "1.4.1"
```

Comment:
major.minor should be sufficient here, to allow for patches, e.g.
once_cell = "1.4"

Reply:
I see, personally I always use "^1.4.1"  https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#caret-requirements but those seem somewhat unpopular for libs so now I used this mix which as you say is not ideal :D